restconf_update_ipaddress/updateip.py

```diff
@@ -26,49 +27,43 @@
 import sys
 from argparse import ArgumentParser
 from collections import OrderedDict
+from getpass import getpass
 import urllib3
 
 # Disable SSL Warnings
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
-
-# These variables target the RESTCONF Always-On Sandbox hosted by Cisco DevNet
-HOST = 'ios-xe-mgmt.cisco.com'
-PORT = '443'
-USER = 'root'
-PASS = 'D_Vay!_10&'
-
 # Identifies the interface on the device used for management access
 # Used to ensure the script isn't used to update the IP leveraged to manage device
 MANAGEMENT_INTERFACE = "GigabitEthernet1"
```

Comment:
I know you left this alone, but this would be another thing to make into a parameter...

Reply:
done